.gitignore

```
@@ -1,3 +1,4 @@
 *.swp
 TODO
+prometheus-meta-operator
```

Only ignore file at repository root

```suggestion
/prometheus-meta-operator
```